src/com/rakovets/course/challenge/classes/TimeTest.java

```diff
@@ -0,0 +1,19 @@
+package com.rakovets.course.challenge.classes;
+
+public class TimeTest {
+    public static void main(String[] args) {
+        Time testingTime = new Time(15, 15, 15);
+        System.out.println("Getting hours: " + testingTime.getHours());
+        System.out.println("Getting minutes: " + testingTime.getMinutes());
+        System.out.println("Getting seconds: " + testingTime.getSeconds());
+        System.out.println("Getting total seconds: " + testingTime.getTotalSeconds());
+        System.out.println("Now i am going to set different time, namely: 22h, 21m, 20s");
+        testingTime.setHours(22);
+        testingTime.setMinutes(21);
+        testingTime.setSeconds(20);
+        System.out.println("Getting hours: " + testingTime.getHours());
+        System.out.println("Getting minutes: " + testingTime.getMinutes());
+        System.out.println("Getting seconds: " + testingTime.getSeconds());
+        System.out.println("Getting total seconds: " + testingTime.getTotalSeconds());
+    }
+}
```

Comment:
I've learned the combination ctrl+alt+l so I guess you'll see everything in green. sorry for that too. I am going to put comments to the places which I should have altered.